frontend/src/components/eam-configurate-group/common/constants.ts

```diff
@@ -0,0 +1,9 @@
+import { EAMGroupCreateRequestDto } from 'common/types/types';
+
+const DEFAULT_GROUP_PAYLOAD: EAMGroupCreateRequestDto = {
+  name: '',
+  tenantId: '',
```

Comment:
```suggestion
```
it shouldn't be in the frontend validation. add it while sending to the backend (in the action)